### PR TITLE
Merge sev repo into the enarx repo

### PR DESCRIPTION
**Pull request by npmccallum**
@steveeJ @mbestavros @lkatalin @haraldh @whitebrandy

There are only two "new" commits in this PR.

1. The merge commit which pulls in the sev repo's history.
2. The workspace commit which adds the sev crate to the workspace.

### CODE REVIEW

**Comment by npmccallum** [comment]
Due to the inherent merge commit in this PR which causes rebasing this branch to be incredibly annoying, I'm putting a freeze on all other merges until this PR is merged.

Please review this PR ASAP.

@lkatalin @steveeJ @haraldh @mbestavros

File: sev/.travis.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  fast_finish: true
+install:
+  - rustup component add rustfmt
+  - cargo install --force cargo-audit
+  - cargo generate-lockfile
+script:
+  - cargo fmt -- --check
+  - cargo audit
+  - cargo test --verbose
+  - cargo test --verbose --features=openssl
```
Comment (mbestavros) [inline sev/.travis.yml]: Enarx already has a Travis config, and this one looks mostly the same. Do we plan on amending this to complement the main config, or will the main config replace this one entirely?

Comment (mbestavros) [inline sev/.travis.yml]: Enarx already has a Travis config, and this one looks mostly the same. Do we plan on amending this to complement the main config, or will the main config replace this one entirely?

Comment (npmccallum) [reply sev/.travis.yml]: @mbestavros Can you please submit a follow up PR for this? Thanks!

**Pull request merged**